tests/ProgramsTest.hs

```diff
@@ -50,16 +52,24 @@ do_test = do
 
   withSystemTempDirectory "c-h-programs-test" $ \tmpdir -> do
 
+  let ext = case System.Info.os of
+              "mingw32" -> "exe"
+              _         -> ""
```

Comment:
That's what [exeExtension](https://hackage.haskell.org/package/directory-1.3.6.1/docs/System-Directory.html#v:exeExtension) is for.

Reply:
Sure, thanks